lib/fetch/formdata.js

```diff
@@ -61,7 +61,13 @@ class FormData {
 
     // The delete(name) method steps are to remove all entries whose name
     // is name from this’s entry list.
-    this[kState] = this[kState].filter(entry => entry.name !== name)
+    const state = this[kState]
+    for (let i = 0; i < state.length; ++i) {
```

Comment:
to give it an additional boost, you should also extract `state.length`
the same applies in all the other loops

Reply:
This only makes sense for large arrays.
Also, since the length of the array is changed, it is necessary to decrement the length of the array, making the code difficult to read.